src/main/java/com/iota/iri/storage/FileExportProvider.java

```diff
@@ -75,7 +72,9 @@ public boolean update(Persistable model, Indexable index, String item) throws Ex
                 } catch (Exception e) {
                     log.error("Transaction load failed. ", e);
                 } finally {
-
+                    if (writer != null) {
+                        writer.close();
```

Comment:
Since this project uses Apache Commons IO, you might want to take a look at `IOUtils.closeQuietly(Closeable closeable)` instead of doing this.

This `writer.close()` could throw an exception too.

I suggest you try here:

```java
} finally {
    IOUtils.closeQuietly(writer);
}
```

Reply:
thanks, I'll change that